sql-cli/sql_cli/macros/load_file.py.jinja2

```diff
@@ -8,57 +8,44 @@
           contains args such as `_name` which cannot be set,
           causing `input_file=File(_name="...", ...)` to fail when parsed by python. #}
     input_file=File(
-        path="{{ yaml_content.input_file.path }}",
-        {%- if "conn_id" in yaml_content.input_file %}
-        conn_id="{{ yaml_content.input_file.conn_id }}",
+        {%- for param_k, param_v in yaml_content.input_file.items() %}
+        {%- if param_k in ["path", "conn_id", "normalize_config", "is_dataframe"] %}
```

Comment:
`path` is required param for `File`. So I believe we should have this outside the `if` so that we raise an error while generating the DAG if the user has not specified `path`.